Assets/Scripts/Game/Entities/PlayerEntity.cs

```diff
@@ -1404,7 +1404,7 @@ public void RegionPowerAndConditionsUpdate(bool updateConditions)
                         }
 
                         // Chance for new ruler
-                        if ((factionData.FactionDict[key].flags & 0x10) == 0)
+                        if (factionData.GetFlag(key, FactionFile.Flags.RulerImmune))
                         {
```

Comment:
This flag check seems reversed. The code in this if() should run if the 0x10 flag is not set, not if it is.

Reply:
Crap, sorry mate. You're quite right, glad you spotted that! Will fix asap.